WTWSMS/decisions/ACR_hellenic_decisions.txt

```diff
@@ -34,12 +34,10 @@ decisions = {
 			}
 			religion_authority = { modifier = pontifex_restored }
 			hidden_tooltip = {
-				narrative_event = { id = ancrel.2104 }
+				narrative_event = { id = ancrel.2104 } #missing?
```

Comment:
I think I added it in commit bfec9e41b16989063161a14f28a825939ecf5e96.

Reply:
done